libs/feature/search/src/lib/state/effects.ts

```diff
@@ -41,14 +42,18 @@ import {
   UPDATE_FILTERS,
   UPDATE_REQUEST_AGGREGATION_TERM,
   UpdateRequestAggregationTerm,
-  SET_FAVORITES_ONLY,
-  SetFilters,
 } from './actions'
 import { SearchState, SearchStateSearch } from './reducer'
 import { getSearchStateSearch } from './selectors'
 import { HttpErrorResponse } from '@angular/common/http'
 import { switchMapWithSearchId } from '../utils/operators/search.operator'
 import { FavoritesService } from '../favorites/favorites.service'
+import { Geometry } from 'geojson'
+
+// this geometry will be used to filter & boost results accordingly
+export const FILTER_GEOMETRY = new InjectionToken<Promise<Geometry>>(
```

Comment:
wonder if it's good to declare the injection token right here.
Might be better to declare it in the module which the file depends on, it will be more readable about "what you can override as an injection token for this module", wdyt ?

Reply:
I agree! good point